scripts/misc/species_list.pl

```diff
@@ -122,7 +122,7 @@ =head1 CONTACT
   # loop over databases
   while (my ($dbname) = $sth->fetchrow_array) {
     next if ($dbname !~ /^[a-z]+_[a-z]+_variation_\d+_\d+$/i);
-    next if ($dbname =~ /^master_schema/ || $dbname =~ /^homo_sapiens_variation_\d+_37$/ || $dbname =~ /private/);
+    next if ($dbname =~ /^(master_schema|drosophila|saccharomyces)/ || $dbname =~ /^homo_sapiens_variation_\d+_37$/ || $dbname =~ /private/);
```

Comment:
drosophila variation database discontinued from e96
http://www.ensembl.info/2019/02/07/whats-coming-in-ensembl-96-ensembl-genomes-43/